integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -258,6 +258,52 @@ async def get_repository(self, repository_id: str) -> dict[Any, Any]:
         repository_data = response.json()
         return repository_data
 
+    async def get_columns(self) -> AsyncGenerator[list[dict[str, Any]], None]:
+        async for boards in self.get_boards_in_organization():
+            for board in boards:
```

Comment:
I like this implementation. looks very clean- however- This requires tests that will make sure this acts as we know it should when stuff changes down the road

Reply:
Done